test/e2e/Tests/Tests/ErrorHandlingTests.cs

```diff
@@ -110,6 +118,7 @@ public async Task OrchestratorWithRetriedActivityException_ShouldSucceed()
 
     [Fact]
     [Trait("MSSQL", "Skip")] // Durable Entities are not supported in MSSQL/Dotnet Isolated, see https://github.com/microsoft/durabletask-mssql/issues/205
+    [Trait("DTS", "Skip")] // DTS will fail this test becasue the exception  type is `InvalidOperationException` instead of `EntityOperationFailedException`
```

Comment:
Can we open a bug for this? It would be better if all backends return a consistent exception type so that customer code doesn't need to be changed when switching to DTS.